block-lexical-variables/test/index.js

```diff
@@ -8,7 +8,7 @@
  * @fileoverview Block test.
  */
 
-import * as Blockly from 'blockly/core';
+import * as Blockly from 'blockly';
```

Comment:
See my comment about this in PR #64

Reply:
@mark-friedman i don't see any comments there yet

Reply:
Sorry, I needed to click another button. Should be there now.

Reply:
ok this one should be good to go now too